gruntfile.js

```diff
@@ -228,6 +228,21 @@ module.exports = function (grunt) {
                 }
             }
         },
+        copy: {
+            commonBundleToTypes: {
+                expand: true,
+                cwd: 'AppInsightsCommon/amd/bundle',
+                src: './**/*.d.ts',
+                dest: 'AppInsightsCommon/amd/types/'
+            }
+        },
+        rename: {
+            commonBundleToTypes: {
+                files: [
+                    {src: 'AppInsightsCommon/amd/types/applicationinsights-common.d.ts', dest: 'AppInsightsCommon/amd/types/index.d.ts'}
```

Comment:
how are we going to share this for now with Jerry?

Reply:
I included a package.json in the destination directory for publishing the types. I've already published v0.0.1